api/src/test/java/org/openmrs/module/sync/SyncOrderSetAttributeTest.java

```diff
@@ -56,7 +63,7 @@ public void runOnParent() throws Exception {
             }
         });
     }
-
+    @Disabled("Disabled until Sync is able to support properties of OpenMRS Objects that are Lists")
```

Comment:
Thanks @mseaton ! I've used the **Disabled** annotation to ignore the test for now.